python/lsst/jointcal/projectionHandler.cc

```diff
@@ -0,0 +1,54 @@
+/*
+ * LSST Data Management System
```

Comment:
Assuming you mean (the template package)[https://github.com/lsst/templates]: it wasn't clear to me that that was up-to-date with our current copyright standard (per RFC-45, or even earlier discussion). Given that RFC-45 is adopted, I think I'll wait on its implementation before changing all the copyright headers.